packages/nhost-js/src/core/nhost-client.ts

```diff
@@ -6,9 +6,10 @@ import { NhostGraphqlClient } from '../clients/graphql'
 
 export interface NhostClientConstructorParams extends Omit<NhostAuthConstructorParams, 'url'> {
   /**
-   * Nhost backend URL.
+   * Nhost backend URL and Region.
```

Comment:
Each option requires its own annotation. This one will go to `backendUrl` while there will be no information about `region` in the generated and inline TS documentation